sentry/src/routes/channel.rs

```diff
@@ -846,4 +859,195 @@ mod test {
             balances.spenders.get(&CREATOR),
         );
     }
+
+    #[tokio::test]
+    async fn get_channels_list() {
+        let mut app = setup_dummy_app().await;
+        app.config.channels_find_limit = 2;
+        let init_chain = GANACHE_1337.clone();
+
+        let mut config = GANACHE_CONFIG.clone();
+
+        // Assert that the Ganache chain exist in the configuration
+        let mut config_chain = config
+            .chains
+            .values_mut()
+            .find(|chain_info| chain_info.chain.chain_id == init_chain.chain_id)
+            .expect("Should find Ganache chain in the configuration");
+
+        // override the chain to use the outpace & sweeper addresses that were just deployed
+        config_chain.chain = init_chain.clone();
+
+        let channel = Channel {
+            leader: IDS[&LEADER],
+            follower: IDS[&FOLLOWER],
+            guardian: *GUARDIAN,
+            token: config_chain
+                .tokens
+                .get("Mocked TOKEN")
+                .expect("Should get token")
+                .address,
+            nonce: Nonce::from(987_654_321_u32),
+        };
+        insert_channel(&app.pool, channel)
+            .await
+            .expect("should insert");
+        let channel_other_token = Channel {
```

Comment:
This is actually other Chain too. We should put emphasis on the Multi-chain instead of multi-token in this test